spec/core.md

```diff
@@ -89,26 +89,27 @@ A GET on that URL MUST return the DID document.
 
 The location of the [[ref: KERI event stream]] is determined by transforming the previous URL as follows:
 
-- Replace the trailing "`/did.json`" with "`/did.keri`".
+- Replace the trailing "`/did.json`" with "`/keri.cesr`".
 
 A GET on that URL MUST return the [[ref: KERI event stream]] for the AID in the `did:webs` identifier.
+The [[ref: KERI event stream]] MUST be a CESR formated stream with media type of application/cesr and the KERI events must be verifiable using the KERI rules.
```

Comment:
I think it is confusing to see “stream” in this part of the sentence `MUST be a CESR formatted stream…`. The GET will return a file containing a the stream of KERI events. I’m OK with the earlier use of “stream”, but think that should be “file”.

Also typo — should be “CESR-formatted"

Reply:
Fixed with commit 8d358e8ca91ea46bf7400ae4166bedbfb3abaade

Reply:
Good stuff.  I think this push to call it a “resource” and not a file is unnecessary.  On the resolver side, anything you GET is a file.

On the controller side — how the web server serves what becomes the file on the resolver side is up to the controller. The controller can put a static file there, can dynamically serve the content from a database, can use a CDN to distribute the content — whatever it wants. I believe that is implied by the web-based nature of the DID Method.

However, in the end, the resolver gets a static file from a point in time to process.

That said, I’m good with the avoidance of the topic here.

Reply:
I think in the URI and HTTP world, the term "resource" is a bit more common for the thing that you dereference and retrieve over port 80/443. If I remember correcly, we switched from the term "file" to "resource" to avoid the impression that it had to be a static file on a dumb web server.